src/SplitButton/index.scss

```diff
@@ -51,12 +51,6 @@
   width: rem(40px);
 }
 
-.nds-splitButton-action--primary,
-.nds-splitButton-action--secondary,
-.nds-splitButton-action--tonal {
-  @extend %btn-kind-base;
-}
-
```

Comment:
This placeholder is no longer necessary, as the `sizes` placeholders now contain the necessary styles that were being applied here.